### ISSUE TITLE
Check if the finished properly

### ISSUE THREAD

**Issue by ondracka** [ondracka/nomad-parser-openmx#17]
The .out file should end with timing section. If we have it, it means the calculation finished properly. Checking for the last line is probably enough.
```

***********************************************************
***********************************************************
               Computational Time (second)                 
***********************************************************
***********************************************************

   Elapsed.Time.       340.525

                               Min_ID   Min_Time       Max_ID   Max_Time
   Total Computational Time =     5      340.518          0      340.525
   readfile                 =    21        3.073         14        3.078
   truncation               =    13       16.575          0       18.765
   MD_pac                   =     0        0.656         23        1.229
   OutData                  =     7        0.000          0        0.003
   DFT                      =     0      317.266         13      318.994

*** In DFT ***

   Set_OLP_Kin              =    19        4.787          1       11.846
   Set_Nonlocal             =     1        8.117         19       15.180
   Set_ProExpn_VNA          =    20       27.576          1       30.713
   Set_Hamiltonian          =     5       85.145         15       85.149
   Poisson                  =     0       42.529         11       42.551
   Diagonalization          =    22        7.994          5        8.023
   Mixing_DM                =     4        0.173         22        0.174
   Force                    =    23       20.457          0       20.457
   Total_Energy             =    23        6.892          0        7.349
   Set_Aden_Grid            =    22        1.645         14        6.754
   Set_Orbitals_Grid        =    12        0.000          1       10.965
   Set_Density_Grid         =    19        2.885          1       53.329
   RestartFileDFT           =     2       26.904         23       39.549
   Mulliken_Charge          =    10        0.556         19        0.557
   FFT(2D)_Density          =     0        0.000          0        0.000
   Others                   =     1        4.548         20       72.802
```
If we see this complete section we should set `section_run.run_clean_end = True.` and False otherwise. I'll add some testcase which exercises the False incomplete out file scenario.

**Comment by Mdmorshadurrahman** [comment]
regarding this issue, when I tried to run before commit, i get the error in manual pytest like this
![image](https://user-images.githubusercontent.com/65698541/121299356-9490ea80-c8f5-11eb-9a9b-4f49b2650317.png)
it was working fine since the last commit. what kind of error is it?

**Comment by ondracka** [comment]
This means the method.smearing_width is currently unitless, this is IMO a bug and should be fixed upstream. I've reported this in https://github.com/nomad-coe/nomad/issues/19 however for now just don't use .magnitude.

**Comment by Mdmorshadurrahman** [comment]
> The .out file should end with timing section. If we have it, it means the calculation finished properly. Checking for the last line is probably enough.
> 
> ```
> 
> ***********************************************************
> ***********************************************************
>                Computational Time (second)                 
> ***********************************************************
> ***********************************************************
> 
>    Elapsed.Time.       340.525
> 
>                                Min_ID   Min_Time       Max_ID   Max_Time
>    Total Computational Time =     5      340.518          0      340.525
>    readfile                 =    21        3.073         14        3.078
>    truncation               =    13       16.575          0       18.765
>    MD_pac                   =     0        0.656         23        1.229
>    OutData                  =     7        0.000          0        0.003
>    DFT                      =     0      317.266         13      318.994
> 
> *** In DFT ***
> 
>    Set_OLP_Kin              =    19        4.787          1       11.846
>    Set_Nonlocal             =     1        8.117         19       15.180
>    Set_ProExpn_VNA          =    20       27.576          1       30.713
>    Set_Hamiltonian          =     5       85.145         15       85.149
>    Poisson                  =     0       42.529         11       42.551
>    Diagonalization          =    22        7.994          5        8.023
>    Mixing_DM                =     4        0.173         22        0.174
>    Force                    =    23       20.457          0       20.457
>    Total_Energy             =    23        6.892          0        7.349
>    Set_Aden_Grid            =    22        1.645         14        6.754
>    Set_Orbitals_Grid        =    12        0.000          1       10.965
>    Set_Density_Grid         =    19        2.885          1       53.329
>    RestartFileDFT           =     2       26.904         23       39.549
>    Mulliken_Charge          =    10        0.556         19        0.557
>    FFT(2D)_Density          =     0        0.000          0        0.000
>    Others                   =     1        4.548         20       72.802
> ```
> 
> If we see this complete section we should set `section_run.run_clean_end = True.` and False otherwise. I'll add some testcase which exercises the False incomplete out file scenario.

what about the test case that you mentioned u will add some.

**Comment by ondracka** [comment]
> what about the test case that you mentioned u will add some.

Yeah, I'm now not sure how it actually works, it seems the .out file is produced only at the end of the calculations, so I can't find a partial out file right now. I have some old cases which seem to be incomplete, but they are several hundreds of MB big, so not something we would want to have in git.

Just test the run_clean_end = True scenario for now.

**Comment by ondracka** [comment]
Fixed with c67f954
